cpp/tests/interop/arrow_utils.hpp

```diff
@@ -42,7 +44,7 @@ get_arrow_array(std::vector<T> const& data, std::vector<uint8_t> const& mask = {
   CUDF_EXPECTS(buff_builder.Finish(&data_buffer).ok(), "Failed to allocate buffer");
 
   std::shared_ptr<arrow::Buffer> mask_buffer =
-    mask.size() > 0 ? arrow::BitUtil::BytesToBits(mask).ValueOrDie() : nullptr;
+    mask.size() > 0 ? arrow::internal::BytesToBits(mask).ValueOrDie() : nullptr;
```

Comment:
Is it cool that we're using internal here? I don't know what Arrow's policies are. :man_shrugging:

Reply:
It compiles and links, so I assume it's ok? This is also only part of the tests, but we could vendor this function if we need to.